### ISSUE TITLE
Homepage Search Bar Component

### ISSUE THREAD

**Issue by hawkticehurst** [CouncilDataProject/cdp-frontend#37]
Implement a homepage search bar component!

![image](https://user-images.githubusercontent.com/39639992/110741853-36674500-81ea-11eb-90aa-b84e62de95e3.png)

Please reference the UX Research on [homepage recommendations](https://docs.google.com/presentation/d/15rkic20QV6GU0_nL-8zHIgcw8-o1bCQIiYMsY2j6qFg/edit#slide=id.g8bf3178c59_0_43).

@JacksonMaxfield @tohuynh Any additional thoughts and comments for a future implementor?

**Comment by JacksonMaxfield** [comment]
Yea I think I may break with the UX teams pure recommendations here a tiny bit.

I think we _should_ have filters on the home page but I think they should be separate from the search bar (as we are already doing with how the components are structured).

One of the goals of the project, as with most plain text search systems is that: "Search should be so good that filters aren't needed for the majority case"

In that way, I think the search bar should be entirely separate and the container for these objects should have:

Search bar spanning the whole container
Filter options below the search bar, greyed out, or some visual dampening if that makes sense

Again, the entire thing should be under mozilla protocol styling

**Comment by JacksonMaxfield** [comment]
That may have been too many comments but search bar component :+1: 

Container that combines all the components of search + filters for landing page :+1:

**Comment by hawkticehurst** [comment]
Additionally, please note that this component library uses [Mozilla Protocol](https://protocol.mozilla.org/) styling so please reference their documentation as this component is being developed. Feel free to literally copy and paste one of the templates that fit closest to the UX notes and then adapt it from there.

There isn't an explicit search bar component, but an implementor could potentially use some of the form input elements as a base component.

https://protocol.mozilla.org/patterns/atoms/forms.html

**Comment by tohuynh** [comment]
I agree with removing filters from the search bar (and move it below) to keep it consistent with other popular search bars.
